### ISSUE TITLE
Model Rho head (ReLU) is incompatible with standard_scaling=True

### ISSUE THREAD

**Issue by GravityBoi** [ratschlab/DeepSpot2Cell#3]
The dataloader.py includes an option for standard_scaling=True. If enabled, this normalizes targets to have zero mean (introducing negative values).

However, the Rho module in model.py ends with nn.ReLU(), and the training step includes .clamp_(min=0.0).

While the paper utilizes log1p (which is non-negative), the repository offers standard_scaling. If a user enables Standard Scaling, the current architecture forces the negative half of the distribution to zero, preventing convergence.

**Suggested Fix:**
If the intention is to support standard_scaling as implemented in the dataloader, the final ReLU in the Rho class and the manual clamping in the training step should be removed or made conditional.

Additionally, Rho currently ends with nn.Dropout(). Typically, dropout is placed before the final projection layer in regression heads, not after, as it introduces stochastic zeroing of the final predictions during training.
```Python

# Current model.py
self.model = nn.Sequential(
    # ...
    nn.Linear(input_size, output_size),
    nn.ReLU(inplace=True), # Incompatible with Standard Scaling
    nn.Dropout(p=0.1),     # Unconventional for final output
)
```
